calender/calendar_tui.py

```
@@ -2180,7 +2218,7 @@ async def _reload_event_day(self, event_date):
 
             self.debug_log(f"Reloading events for {event_date}")
             await self._fetch_week_range(start_of_day, end_of_day, replace=True)
-            self._find_current_event()
+            self._restore_cursor(saved_row, saved_event_id)
             self.draw()
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)